src/buildblock/ProjDataInfoCylindricalNoArcCorr.cxx

```diff
@@ -489,8 +490,9 @@ find_cartesian_coordinates_given_scanner_coordinates (CartesianCoordinate3D<floa
   LORInCylinderCoordinates<float> cyl_coords(get_scanner_ptr()->get_effective_ring_radius());
   cyl_coords.p1().psi() = static_cast<float>((2.*_PI/num_detectors_per_ring)*(det1));
   cyl_coords.p2().psi() = static_cast<float>((2.*_PI/num_detectors_per_ring)*(det2));
-  cyl_coords.p1().z() = Ring_A*get_scanner_ptr()->get_ring_spacing();
+  cyl_coords.p1().z() = Ring_A*get_scanner_ptr()->get_ring_spacing(); 
   cyl_coords.p2().z() = Ring_B*get_scanner_ptr()->get_ring_spacing();
+  // ORIGINTODO: ^
```

Comment:
I don't think any changes needed as remains in same coordinate system

Reply:
Yep